Build fails on illumos with GCC 14+

Build fails on illumos with this error:
```
OS.c: In function 'OS_get_table':
OS.c:121:5: error: implicit declaration of function 'bless_into_proc' [-Wimplicit-function-declaration]
  121 |     bless_into_proc( Format,
      |     ^~~~~~~~~~~~~~~
```

--- a/os/Solaris.h
+++ b/os/Solaris.h
@@ -31,6 +31,8 @@
 #define STOP   "stop"
 #define ONPROC "onprocessor" 
 
+extern void bless_into_proc(char* format, char** fields, ...);
+
 /* Solaris is an all-or-nothing deal, all this stuff comes out of 
    one structure, so we don't need to dick around with the format much */
 #if defined(PROC_FS)

